lib/pages/chat/chat_event_list.dart

```diff
@@ -167,3 +174,45 @@ class ChatEventList extends StatelessWidget {
     }
   }
 }
+
+class SelectionTextContainer extends StatelessWidget {
+  final Widget child;
+
+  final FocusNode focusNode;
+
+  final ChatController chatController;
+
+  const SelectionTextContainer({
+    super.key,
+    required this.child,
+    required this.focusNode,
+    required this.chatController,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    if (!PlatformInfos.isWeb) {
+      return child;
+    }
+
+    return CallbackShortcuts(
+      bindings: {
+        SingleActivator(
+          LogicalKeyboardKey.keyC,
+          meta: PlatformInfos.isMacKeyboardPlatform,
```

Comment:
should create another class to reuse this check, you used it in some cases

Reply:
It's different logic, so it's hard to find anything to reused here

Reply:
maybe defined const `SingleActivator` to a class

Reply:
like that
```
SingleActivator.adaptiveControl(LogicalKeyboardKey.keyC)
```

Reply:
but some shortcut have ctrl, command, so if change like you, it's should be like before is better

Reply:
not only control,cmd also have shift, and future may be shift+cmd, .... option, ...

Reply:
then we should keep it like before